docs/030-policies/security.md

```diff
@@ -129,7 +129,7 @@ CivicActions requires unique, strong passwords for every service that you log in
 
 Use both a unique, strong password (or multi-word passphrase) _and_ two-factor authentication (TFA or 2FA) to secure your password manager.
 
-CivicActions supplies all employees with a YubiKey for two-factor authentication, and we require two-factor authentication for access to your password manager, the CivicActions GSuite of applications, and OAuth-authenticated apps such as GitHub and GitLab. You can also use second factor authentication apps such as Authy and Duo, and in fact, for backup reasons we suggest using more than one second factor.
+CivicActions supplies all employees with a YubiKey for two-factor authentication, and we require two-factor authentication for access to your password manager, the CivicActions GSuite of applications, and OATH-authenticated apps such as GitHub and GitLab. You can also use second factor authentication apps such as Authy and Duo, and in fact, for backup reasons we suggest using more than one second factor.
```

Comment:
I messed this up myself in one of my first weeks here. :facepalm:

Reply:
@billwanjohi I don't think most team members would know what OAuth or OATH is, and I am not sure it matters here.
I think we could rephrase this whole sentence to be much more concise: "access to your password manager, CivicActions GSuite, GitHub and GitLab"?

Reply:
Everything else in this PR looks great - thank you :)

Reply:
OK, as suggested, I trimmed that.